#17: Documented GitHub authentication details (#23)
exasol/release-robot: doc/user_guide/user_guide.md

### Rules for Release on GitHub

Prerequisites:

1. You need a GitHub account;
2. You need to create a GitHub OAuth token. 

>    For that, go to `Settings` -> `Developer Settings` -> `Personal access tokens` -> `Generate new token`. 

Reviewer: ```suggestion
   For that, go to `Settings` &rarr; `Developer Settings` &rarr; `Personal access tokens` &rarr; `Generate new token`. 
```

1. Download the latest available [release](https://github.com/exasol/release-robot/releases) of Release Robot.

1. (Optional) Place a file with credentials  in your home directory: `~/.release-robot/credentials`.

>     We recommend restricting access to this file in security purposes: `chmod u-wx,g-rwx,o-rwx "$HOME"/.release-robot/credentials`.

Reviewer: ```suggestion
    We recommend restricting access to this file for security purposes:
    ```bash
    chmod u-wx,g-rwx,o-rwx "$HOME"/.release-robot/credentials
    ````
```
